pyproject.toml

```
@@ -66,6 +66,7 @@ python = "^3.6"
 coloredlogs = "^10.0"
 sanic = "^19.12.2"
 sanic-cors = "^0.10.0b1"
+requests = "^2.23"
```

I think we can allow older ones, too, right? How about
```suggestion
requests = "^2.0"
```